pkg/test/framework/components/opentelemetry/kube.go

```diff
@@ -133,7 +133,7 @@ func install(ctx resource.Context, ns string) error {
 func installServiceEntry(cluster resource.Cluster, ctx resource.Context, ns, ingressAddr string) error {
 	// Setup remote access to zipkin in cluster
 	yaml := strings.ReplaceAll(remoteOtelEntry, "{INGRESS_DOMAIN}", ingressAddr)
-	err := ctx.Config().ApplyYAMLInCluster(cluster, ns, yaml)
```

Comment:
ApplyYAMLInCluster did NOT actually do what it said. I am going to make it retain the current behvaior (apply to all clusters) and file a bug that the tracing tests are not really multicluster